src/Components/Admin/Rooms/roomCard.js

```diff
@@ -81,7 +81,9 @@ export default function RoomCard({ data, handleEditClick, handleDeleteClick }) {
             >
                 <img
                     className={classes.cardImage}
-                    src={data.image ? data.image : defaultImage}
+                    src={
+                        data.screenshot_url ? data.screenshot_url : defaultImage
```

Comment:
You are saving the full screenshot URL in the backend, instead of just the key (see backend PR comment: https://github.com/uwblueprint/cdc-backend/pull/149#discussion_r673629733). So once you fix that in the backend PR, I believe you will have to change the screenshot_url here to prepend the s3 prefix.